coco/coco.go

```diff
@@ -0,0 +1,91 @@
+package coco
+
+import (
+	"os"
+	"fmt"
+
+	"text/template"
+
+	gen "github.com/ibm-hyper-protect/contract-go/common/general"
+)
+const (
+	emptyParameterErrStatement = "required parameter is empty"
+)
+
+const tomlTemplate = `
+algorithm = "sha384"
+version = "0.1.0"
+
+[data]
+"contract.yaml" = '''{{ .Encrypted_contract }}'''
+`
+
+// HpccGzippedInitdata - Function to generate gzipped initdata value. 
+func HpccGzippedInitdata(contract string) (string , error) {
+
+	if gen.CheckIfEmpty(contract) {
+		return "", fmt.Errorf(emptyParameterErrStatement)
+	}
+
+	tomlFile, err := generateTomlFile(contract)
+	if err != nil {
+		return "", err
+	}
+
+	_, err = os.Stat(tomlFile)
+	if err != nil {
+		return "", fmt.Errorf("Failed to check the toml file")
```

Comment:
I'm not sure about the style guidelines on this repo, but go standards are that errors should start with lower case (see https://go.dev/wiki/CodeReviewComments#error-strings)